src/frontend/components/DashboardVideoLiveStopButton/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback, useState } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { Redirect } from 'react-router-dom';
+
+import { stopLive } from '../../data/sideEffects/stopLive';
+import { useVideo } from '../../data/stores/useVideo';
+import { Video } from '../../types/tracks';
+import { Nullable } from '../../utils/types';
+import { DashboardButton } from '../DashboardPaneButtons';
+import { ERROR_COMPONENT_ROUTE } from '../ErrorComponent/route';
+import { Loader } from '../Loader';
+
+const messages = defineMessages({
+  stopLive: {
+    defaultMessage: 'stop streaming',
+    description: 'Stop a video streaming.',
+    id: 'components.DashboardVideoLiveStopButton.startLive',
+  },
+});
+
+type startLiveStatus = 'pending' | 'error';
+
+interface DashboardVideoLiveStartButtonProps {
+  video: Video;
+}
+
+export const DashboardVideoLiveStopButton = ({
+  video,
+}: DashboardVideoLiveStartButtonProps) => {
+  const [status, setStatus] = useState<Nullable<startLiveStatus>>(null);
+  const { updateVideo } = useVideo((state) => ({
+    updateVideo: state.addResource,
+  }));
+
+  const stopLiveAction = useCallback(async () => {
+    setStatus('pending');
+    try {
+      const updatedVideo = await stopLive(video);
+      updateVideo(updatedVideo);
```

Comment:
Does Status stay pending after live stopped?

Reply:
Yes we don't need to change it, after updating video this component will be destroyed.